shared/common-adapters/custom-emoji.native.tsx

```diff
@@ -1,18 +1,29 @@
 import * as React from 'react'
+import * as Styles from '../styles'
 import {Props} from './custom-emoji'
 import {FastImage} from './native-image.native'
+import {Box2} from './box'
 
 const CustomEmoji = (props: Props) => {
   const {size, src} = props
   return (
-    <FastImage
-      source={{uri: src}}
-      style={{
-        height: size,
-        width: size,
-      }}
-    />
+    <Box2 direction="vertical" style={styles.container}>
```

Comment:
This `Box2` needs an explicit height set to not break android, should be able to reuse the same width and height style from the image!

Reply:
Updated! It still looks good on iOS. I'm still working on testing on Android.